apps/nova/src/App.tsx

```diff
@@ -26,6 +26,7 @@ import Interviews from "@/pages/Interviews";
 import Invoices from "@/pages/Invoices";
 import Chat from "@/pages/Chat";
 import { SocketProvider } from "@litespace/luna";
+import ResetPassword from "@/pages/ResetPassword";
```

Comment:
Let's swap these lines such that pages are grouped together. 

```suggestion
import Chat from "@/pages/Chat";
import ResetPassword from "@/pages/ResetPassword";
import { SocketProvider } from "@litespace/luna";
```